build.gradle

```diff
@@ -3,6 +3,7 @@ plugins {
 }
 
 dependencies {
+    implementation group: 'commons-validator', name: 'commons-validator', version: '1.7'
```

Comment:
This works perfectly well, and is pretty straightforward. The approach that is used in most projects however, is to use toml. Check ./gradle/libs.versions.toml